### ISSUE TITLE
Invalid textField names returned by display capabilities

### ISSUE THREAD

**Issue by NicoleYarroch** [smartdevicelink/sdl_hmi#215]
### Bug Report
The `RegisterAppInterface` response `displayCapabilities` `textFields` array contains invalid `name`s. The `name` field should be a `String` but some of the names have `Integer`s. 

##### Reproduction Steps
1. Connect SDL_iOS example app to sdl_core and sdl_hmi
2. Check the `textFields` field returned by the RIA response.

##### Expected Behavior
All `name`s in the `textFields` array should be a valid `TextFieldName` string.

##### Observed Behavior
Some of the `name`s in the `textFields` array are numbers.

##### OS & Version Information
* sdl_hmi version: 5.1.0
* sdl_core version: 5.1.3
* Testing with: SDL_iOS example app (v. 6.3)

**Comment by NicoleYarroch** [comment]
Below is the `textFields` with the invalid `name`s:
``` 
       textFields =         (
                        {
                characterSet = TYPE2SET;
                name = mainField1;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = mainField2;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = mainField3;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = mainField4;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = statusBar;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = mediaClock;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = mediaTrack;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = templateTitle;
                rows = 1;
                width = 100;
            },
                        {
                characterSet = TYPE2SET;
                name = alertText1;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = alertText2;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = alertText3;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = scrollableMessageBody;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = initialInteractionText;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = navigationText1;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = navigationText2;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = ETA;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = totalDistance;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = locationName;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = audioPassThruDisplayText1;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = audioPassThruDisplayText2;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = sliderHeader;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = sliderFooter;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = locationDescription;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = menuName;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = secondaryText;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = tertiaryText;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = 31;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = 32;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = menuTitle;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = addressLines;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = phoneNumber;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = 29;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = 30;
                rows = 1;
                width = 500;
            }
        );
```

**Comment by iCollin** [comment]
Per @jacobkeeler 
> Seems the issue is actually with Core, there are 4 text fields that are defined in the HMI_API, but not the MOBILE_API ("navigationText", "notificationText", "timeToDestination", and "turnText"), so they just get passed as the internal enum values. `turnText` and `notificationText` can be safely removed from the HMI API, but the other two are used at the moment.

**Comment by iCollin** [comment]
Closed with merge of #225
